src/locales/zh_CN/battle.json

```
@@ -32,6 +32,7 @@
   "attackMissed": "没有命中{{pokemonNameWithAffix}}！",
   "attackHitsCount": "击中{{count}}次！",
   "rewardGain": "你获得了\n{{modifierName}}！",
+  "rewardGainCount": "你获得了\n{{count}}个{{modifierName}}!",
```

```suggestion
  "rewardGainCount": "你获得了\n{{count}}个$t(modifierName, {\"count\": {{count}} })!",
```

For possible plural handling

DISCLAIMER: Does `zh_CN` need this kind of plural handling?